templates/artist/includes/artist_detail_past.html

```diff
@@ -4,9 +4,58 @@
     <h2>PAST CAMPAIGNS</h2>
     <ul>
         {% for past_campaign in artist.past_campaigns %}
+        <div class="break">
+            <hr>
+        </div>
+        <div class="past-campaign">
             <li>
-                ${{ past_campaign.amount|intcomma }} {{ past_campaign.project.reason }}: {{ past_campaign.percentage_funded }}% funded [ended {{ past_campaign.end_datetime|timesince }} ago]
+                <h3>${{ past_campaign.amount|intcomma }} {{ past_campaign.project.reason }}: {{ past_campaign.percentage_funded }}% funded</h3>
+                <h4>[ended {{ past_campaign.end_datetime|timesince }} ago]</h4>
+                <!-- Countdown Widget -->
+                <div class="artist-detail-padding">
```

Comment:
I don't think it makes sense to copy and paste the Countdown Widget exactly here. The number of days remaining will always be 0, so that's kind of a useless metric.

It might also not be a great idea to stress that a campaign that didn't get funded raised less than 100% of the goal, which I think the thermometer does stress.